modin/pandas/test/test_io.py

```diff
@@ -253,7 +261,7 @@ def test_read_csv_delimiters(
 
     # Column and Index Locations and Names tests
     @pytest.mark.skipif(
-        Engine.get() != "Python",
+        Engine.get() != "Python" and Backend.get() != "Omnisci",
```

Comment:
```suggestion
        Engine.get() != "Python" or Backend.get() != "Omnisci",
```
shouldn't it be `or`?